src/modules/dashboard/service/dashboardService.ts

```diff
@@ -2568,5 +2582,28 @@ export class DashboardService {
     static convertToExactNativeAmount(amount: number){
         return amount > 0 ? amount / Math.pow(10, nativeTokenDivisibility.value) : 0;
     }
+
+    static displayOtherAsset = async(amount: number, assetId: string, mosaic_id: MosaicId) => {
```

Comment:
why async ?

Reply:
The `await` was commented. This part belongs to the 'linked to namespace' asset, difficult to derive with current dashboard services codebase with the api due to the async and await

Reply:
then it do not need to be async